index.php

```diff
@@ -1,19 +1,29 @@
+<?php
+$nazevParty = "Czechitas";
```

Comment:
Nechtěli jsme to dělat spíš univerzální, jako registrační systém na akce? Byť párty je samozřejmě ta ukázková akce.

Takže `$nazevAkce = "Czechitas Party";`,  `<h1><?php echo $nazevAkce; ?></h1>` atd.?